graf2d/cocoa/src/TGCocoa.mm

```diff
@@ -13,11 +13,7 @@
 
 #include "TGCocoa.h"
 
-// We want to pickup ROOT's glew and not the system OpenGL coming from:
-// ROOTOpenGLView.h ->QuartzWindow.h->Cocoa.h
-// Allowing TU's which include the system GL and then glew (from TGLIncludes)
-// leads to gltypes.h redefinition errors.
-#include "TGLIncludes.h"
+#include <GL/glew.h>
```

Comment:
The comment seemed out of date. The "TGLIncludes.h" header only included `<GL/glew.h>` anyway.